src/PeliasMapkitExtensions.swift

```diff
@@ -144,3 +144,10 @@ extension SearchBoundaryRect {
     self.minLatLong = GeoPoint(latitude: maxCoordinate.latitude, longitude: minCoordinate.longitude)
   }
 }
+
+public extension GeoPoint {
```

Comment:
Do you think there is any way we could reduce the number of structures made to represent a location? Mostly thinking of `TGGeoPoint` and this one.